com.microsoft.mrtk.input/Assets/Input Actions/MRTK Default Input Actions.inputactions

```diff
@@ -1018,26 +1080,70 @@
             ],
             "bindings": [
                 {
-                    "name": "",
-                    "id": "899c9b8b-e94b-486f-a267-3eaf0b8fe703",
+                    "name": "Vector 3 Fallback",
+                    "id": "102eaa91-e56e-4f09-a6a4-7b5d7c0933a6",
+                    "path": "Vector3Fallback",
+                    "interactions": "",
+                    "processors": "",
+                    "groups": "",
+                    "action": "Position",
+                    "isComposite": true,
+                    "isPartOfComposite": false
+                },
+                {
+                    "name": "first",
+                    "id": "766df8af-7f51-4bcc-a415-a24d97dd949c",
                     "path": "*/{gaze}/position",
                     "interactions": "",
                     "processors": "",
-                    "groups": "Generic XR Controller",
+                    "groups": "",
+                    "action": "Position",
+                    "isComposite": false,
+                    "isPartOfComposite": true
+                },
+                {
+                    "name": "second",
+                    "id": "ca8a9a3a-1867-4a6a-b474-91fbe0e43ba1",
+                    "path": "<XRHMD>/centerEyePosition",
```

Comment:
Just double checking that this wasn't unintentionally brought over from my branch! I added these fallbacks for eye gaze to the camera pose in order to test gaze+pinch on a device like Quest that doesn't support the eye gaze interaction profile.

Reply:
This was intentional. This unlocks gazepinch on Quest + Hololens-with-no-eye-calibration!

Reply:
Works for me!